services/brig/test/integration/API/User/Account.hs

```diff
@@ -791,6 +795,73 @@ testMultipleUsers brig = do
     field :: FromJSON a => Key -> Value -> Maybe a
     field f u = u ^? key f >>= maybeFromJSON
 
+testMultipleUsersV3 :: Opt.Opts -> Brig -> Http ()
```

Comment:
Shouldn't this one be called `testMultipleUsers` (as it uses the latest API version, namely v4) and the other one `testMultipleUsersV3` (as it uses the old v3)?